formatters/jdt-core/build.properties

```diff
@@ -1,3 +1,20 @@
+#
+# Copyright 2010-2016. All work is copyrighted to their respective
```

Comment:
We shouldn't put the copyright portion in the license headers. We should put those in the NOTICE file reference in the Apache LICENSE. It's friendlier for downstream projects.